packages/core/src/attestation/Attestation.chain.ts

```diff
@@ -159,25 +128,12 @@ export async function getRevokeTx(
 ): Promise<SubmittableExtrinsic> {
   const api = await BlockchainApiConnection.getConnectionOrConnect()
   log.debug(() => `Revoking attestations with claim hash ${claimHash}`)
-  const authorizationArgName =
-    api.tx.attestation.revoke.meta.args[1].name.toString()
-  switch (authorizationArgName) {
-    case 'maxParentChecks':
-      // uses old attestation authorization
-      return api.tx.attestation.revoke(claimHash, maxParentChecks)
-    case 'authorization':
-      // uses generalized attestation authorization
-      return api.tx.attestation.revoke(
-        claimHash,
-        maxParentChecks
-          ? ({ delegation: { maxChecks: maxParentChecks } } as any) // subjectNodeId parameter is unused on the chain side and therefore omitted
-          : undefined
-      )
-    default:
-      throw new SDKErrors.CodecMismatchError(
-        'Failed to encode call: unknown authorization type'
-      )
-  }
+  return api.tx.attestation.revoke(
+    claimHash,
+    maxParentChecks
+      ? ({ delegation: { maxChecks: maxParentChecks } } as any) // subjectNodeId parameter is unused on the chain side and therefore omitted
```

Comment:
`as any` should not be necessary any more - this should match the autogenerated types